cmd/calyptia/create_core_instance_gcp.go

```diff
@@ -138,11 +149,26 @@ func newCmdCreateCoreInstanceOnGCP(config *config, client gcp.Client) *cobra.Com
 	fs.StringVar(&sshUser, "ssh-user", "", "GCP SSH user to use for the instance. (default is no user)")
 	fs.StringVar(&sshKeyPath, "ssh-key", "", "SSH Key path to use for the instance. (default is no key)")
 	fs.StringVar(&network, "network", "default", "GCP Network")
-	fs.BoolVar(&useTestImages, "use-test-images", false, "Use GCP test images instead of released channel (only for testing/development).")
 
 	return cmd
 }
 
+func getGitHubToken() (string, error) {
+	token := os.Getenv("GITHUB_TOKEN")
```

Comment:
Totally not a fan of reading environment variables inside a function. It's a side-effect that's difficult to catch.

Maybe this whole thing can be done differently... 🤔

Reply:
Like you can hide a flag:

```go
fs.MarkHidden("test-github-token")
```

https://pkg.go.dev/github.com/spf13/pflag#FlagSet.MarkHidden
That will cause the flag to still work, but don't appear on the help.

Reply:
Also, env variables from the CLI should be `CALYPTIA_` prefixed to avoid reading other important stuff from the host.

Reply:
GITHUB_TOKEN is a standard, lets don't move away from it. As for the flag, we can do both, env + mark hidden on this case @gabrielbussolo